lib/game_teams.rb

```diff
@@ -26,9 +26,9 @@ def initialize(params)
     @shots = params[:shots]
     @tackles = params[:tackles]
     @pim = params[:pim]
-    @powerPlayOpportunities = params[:powerPlayOpportunities]
```

Comment:
Here is an example. In the csv file we are referencing, we created a key from the headers called `:powerplayopportunities` but this line was pointing to `:powerPlayOpportunities` ... symbols are case sensitive.

Reply:
Good catch!